misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/LocalDateFeatureAssertionsSpec.kt

```diff
@@ -62,4 +67,38 @@ abstract class LocalDateFeatureAssertionsSpec(
             }
         }
     }
+
+    describeFun("val ${monthFeature.name}") {
+        val monthVal = monthFeature.lambda
+
+        context("LocalDate with month March(3)") {
+            it("toBe(March) holds") {
+                fluent.monthVal().toBe(3)
+            }
+            it("toBe(April) fails") {
+                expect {
+                    fluent.monthVal().toBe(4)
+                }.toThrow<AssertionError> {
+                    messageContains("month: 3" )
```

Comment:
Please use the default translation instead of a hard-coded string.
Introduce a local val above after `fluent`:
```
val monthDescr = DescriptionDateTimeLikeAssertion.MONTH.getDefault()
```
Then this can be changed to
```suggestion
                    messageContains("$monthDescr: 3")
```
Please do the same for the other hard coded `"month"` below and it would be great, if you could do this also for the `"year"` above (must have missed that in a previous review).